File: Java/org/mayope/api/request/ListTreesRequest.java
package org.mayope.api.request;

import java.util;

/**
 * A request for a list of trees.
 */
public class ListTreesRequest extends AuthenticatedRequest {
    private UUID forest;

    public UUID getForest() {
        return forest;
    }

    public void setForest(UUID forest) {
        this.forest = forest;
    }
}

File: CSharp/Mayope.Api/Requests/ListTreesRequest.cs
using System;
using System.ComponentModel.DataAnnotations;

using Newtonsoft.Json;

namespace Mayope.Api.Requests;
{
    /// <summary>
    /// A request for a list of trees.
    /// </summary>
    public class ListTreesRequest : AuthenticatedRequest
    {
        /// <summary>
        /// The UUID of the forest to list the trees in.
        /// </summary>
        [Required]
        [JsonRequired]
        public Guid Forest { get; set; }
    }
}
